source/renderer/app/features/menu-updater/MenuUpdater.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import { inject } from 'mobx-react';
+import type { StatelessFunctionalComponent } from 'react';
+import type { StoresMap } from '../../stores';
+import { useFeature } from '../../utils/mobx-features/hooks';
+import MenuUpdaterFeature from './feature';
+import type { MenuUpdaterProps } from './types';
+
+const MenuUpdater: StatelessFunctionalComponent<MenuUpdaterProps> = ({
```

Comment:
Maybe rename to `MenuUpdaterProvider` so we know this component is required by the app.

Reply:
I think it is common to call component a `Provider` if such component provides something through the react context. However I am happy to rename it if this is our rule for such components.